code/datums/uplink_item.dm

```diff
@@ -278,7 +276,9 @@ var/list/uplink_items = list()
 	reference = "PG"
 	item = /obj/item/clothing/gloves/color/yellow/power
 	cost = 10
-	job = list("Station Engineer","Chief Engineer")
+	job = list(
```

Comment:
It was already done, just inconsistently. Figured the idea was that it should be this way.

Perhaps threaten the dude that did it 'wrong' originally?

Or to write up some actual coding standards for Paradise rather than people having to guess them or come across them by trying to help out, and threatened?